router.py

```diff
@@ -1,8 +1,29 @@
 from fastapi import APIRouter
 
 
+from models import SourceTextLen, SourceText
+##from utils import Utils
+
 api_router = APIRouter()
+generator = pipeline("text-generation", "gpt2")
 
 @api_router.get("/")
 async def root():
     return {"message": "Формализация текста"}
+
+@api_router.post("/generate_len/")
+def generate_len(source: SourceTextLen):
+    """Text generation using user input
+    - **text**: input user text
+    - **text_len**: count of output symbols
+    """
+    return generator.generate_text(source.text, source.text_len)
+
+
+@api_router.post("/generate_100/")
```

Comment:
Better to remove this endpoint. So far it has no sense. We can return it in future if needed.